src/devices/efekta.ts

```diff
@@ -750,6 +750,236 @@ const definitions: DefinitionWithExtend[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ['EFEKTA_Air_Quality_Station'],
+        model: 'EFEKTA_Air_Quality_Station',
+        vendor: 'EFEKTA',
+        description:
+            '[EFEKTA Air Quality Station, PM1.0, PM2.5, PM4.0, PM10, AQ PM2.5 Index, PM Size, CO2, VOC Index, Temperature, \
```

Comment:
Please change the description to just `Air Quality Station`, link can be included on the docs page.

Reply:
Hello, I have made changes, which I will also take into account in subsequent converters.